homeassistant/components/qbittorrent/coordinator.py

```diff
@@ -33,7 +40,21 @@ def __init__(self, hass: HomeAssistant, client: Client) -> None:
         )
 
     async def _async_update_data(self) -> dict[str, Any]:
+        """Async method to update QBittorrent data."""
         try:
             return await self.hass.async_add_executor_job(self.client.sync_main_data)
         except LoginRequired as exc:
-            raise ConfigEntryError("Invalid authentication") from exc
+            raise HomeAssistantError(str(exc)) from exc
```

Comment:
Why did we change the exception in this PR? That looks unrelated to the new services.